packages/grafana-ui/src/components/Gauge/Gauge.tsx

```diff
@@ -119,18 +120,31 @@ export class Gauge extends PureComponent<Props> {
   }
 
   render() {
-    const { height, width } = this.props;
+    const { height, width, value } = this.props;
 
     return (
       <div
         style={{
           height: `${Math.min(height, width * 1.3)}px`,
           width: `${Math.min(width, height * 1.3)}px`,
-          top: '10px',
           margin: 'auto',
+          marginTop: '-8px',
         }}
         ref={element => (this.canvasElement = element)}
-      />
+      >
+        {value.title && (
+          <div
+            className={css(`
+          text-align:center;
+          bottom:-8px;
+          width:100%;
+          position:absolute;
+        `)}
```

Comment:
@dprokop -- I copied the emotion use from your legend PR.  I like it!

But... is there a type safe way to do this?  seems like you can put any strings you want in.  It would be great to have it take a CSSProperties object or something

Reply:
ha -- nevermind, I found it

Reply:
:)